### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by maya**
Brings the issue type to the content following writing lab's style: https://github.com/18F/writing-lab/issues/new

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
 <!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
 
-## Title Line Template: [Website] - [UI component]: [Brief description.]
+## Title Line Template: [Website or UI component]: [Brief description]
 <!-- Use the title line as the title of your pull request, then delete these lines. 
 
 Website: Issues that impact standards.usa.gov look, feel or functionality.
```
Comment (kategarklavs) [inline .github/ISSUE_TEMPLATE.md]: Please add a comma after `feel` (and before or).

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 UI component: Issues that impact the look, feel or functionality of the standards themselves.
 
 -->
 
+## Issue type
+What kind of issue are you working on?
+
+* Website
+* UI component
+* Something else...
```
Comment (kategarklavs) [inline .github/ISSUE_TEMPLATE.md]: Maybe for `Something else`, we could include a parenthetical: (Please add a short description of your issue.)

Comment (maya) [inline .github/ISSUE_TEMPLATE.md]: Or `other`?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (maya) [inline .github/ISSUE_TEMPLATE.md]: Fixed in 766543b.

**Comment by rogeruiz** [comment]
Looks good from here 👍

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (maya) [inline .github/ISSUE_TEMPLATE.md]: Fixed in 22db757.

**Pull request merged**